### ISSUE TITLE
Contribution guide

### ISSUE THREAD

**Issue by milmazz** [telerik/kendo-ui-core#1390]
According to your [contribution guide][1] we can run the tests with `grunt tests` or verify the code style with `grunt jshint`, but those tasks fail because the task directory `build/grunt/tasks` do not exist. Am I doing something wrong or the contribution guide needs to be updated to use `gulp` instead of `grunt`?

```console
$ grunt tests
Loading "debug.js" tasks...ERROR
>> ReferenceError: fs is not defined
>> Tasks directory "build/grunt/tasks" not found.
Loading "Gruntfile.js" tasks...ERROR
>> AssertionError: missing path
Warning: Task "tests" not found. Use --force to continue.

Aborted due to warnings.
```

The following works fine:

```console
$ gulp jshint
[13:47:24] Using gulpfile d:\kendo-ui-core\gulpfile.js
[13:47:24] Starting 'jshint'...
[13:47:34] Finished 'jshint' after 10 s

$ gulp travis
[13:52:19] Using gulpfile d:\kendo-ui-core\gulpfile.js
[13:52:19] Starting 'travis'...
[13:52:19] Starting 'jshint'...
[13:52:30] Finished 'jshint' after 11 s
[13:52:30] Starting 'scripts'...
[13:52:30] Starting 'less'...
[13:52:30] Starting 'css-assets'...
...
```
[1]: https://github.com/telerik/kendo-ui-core/blob/master/CONTRIBUTING.md

**Comment by underlog** [comment]
Hi @milmazz, 

sorry for that - we migrated from grunt to gulp, but some leftovers are still present. We will fix that. Thank you very much!

**Comment by underlog** [comment]
Fixed in c01f44ea857b8b1ad3ef3cb00d8254592190a30e
